definitions/external-service/definition.yml

```diff
@@ -0,0 +1,16 @@
+domain: EXTERNAL
+type: SERVICE
+
+synthesis:
+  rules:
+    # DistributedTraceProcessSummary events
+  - identifier: entity.guid
```

Comment:
This will do nothing on the engine. Since when a guid is present the engine will bypass it.

Is this intended? I'm not sure if we are intending the engine to decide the guid on the telemetry or are we just using the definitions to materialize the entity

Reply:
A "normal" rule will be like
```
- identifier: hostname
  name: hostname
 condition: ...
```

The engine will give you back the guid & type and you stamp that and send downstream

Reply:
> are we just using the definitions to materialize the entity

That is exactly what we're trying to do